mftf/2.3/merging.md

```diff
@@ -270,6 +278,35 @@ The `LogInAsAdminTest` result corresponds to:
 </test>
 ```
 
+## Merge action groups
+
+Merging action groups allows you to extend existing tests by reusing existing action groups, while customizing them for your specific needs.
+
+### Use Case
+
+Here is an action group for selecting `customerGroup` in the `Cart Price Rules` section.
+The controls change drastically in the B2B version, so it was abstracted to an action group.
+This way it could be easily changed if B2B is enabled.
+
+```xml
+<actionGroup name="selectNotLoggedInCustomerGroup">
+    <selectOption selector="{{AdminCartPriceRulesFormSection.customerGroups}}" userInput="NOT LOGGED IN" stepKey="selectCustomerGroup"/>
+</actionGroup>
+
+B2B Merge file
```

Comment:
```suggestion
> B2B merge file
```

Reply:
The > acts as a subtitle of sorts for the code sample.

Reply:
Nice!